x-pack/plugins/task_manager/server/task_runner.ts

```diff
@@ -156,15 +157,23 @@ export class TaskManagerRunner implements TaskRunner {
       taskInstance: this.instance,
     });
 
+    const trans = apm.startTransaction(
+      `taskManager run ${this.instance.taskType}`,
+      'taskManager',
+      this.instance.taskType,
```

Comment:
The `name` and `type` (first two parms ^^^) seem useful, in terms of how the UI ends up making use of them.  The `subtype` and `action` (last two parms) I'm less sure of, as I'm not sure they come up directly in the UI.  Since these end up being other plugins tasks running, seems unlikely we can consistently get more specific per-task info.  And alerting and actions already have alertType and actionType specific task types, which makes for nice grouping.  

Thinking of just removing the last two.

Reply:
resolved